src/wp-includes/blocks/pattern.php

```diff
@@ -48,7 +61,15 @@ function render_block_core_pattern( $attributes ) {
 		$content = gutenberg_serialize_blocks( $blocks );
 	}
 
-	return do_blocks( $content );
+	$seen_refs[ $attributes['slug'] ] = true;
+
+	$content = do_blocks( $content );
+
+	global $wp_embed;
```

Comment:
Need to document the use of this global in the function's docblock. [Ref](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#1-functions-class-methods)

Reply:
cc @dsas

Can you please follow-up with a PR in Gutenberg to address the comments that are related to the changes here

Reply:
Ooops, opened https://github.com/WordPress/gutenberg/pull/58382 🙇‍♂️